lang/java/avro/src/test/java/org/apache/avro/reflect/TestReflectData.java

```diff
@@ -40,6 +41,17 @@
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
 public class TestReflectData {
+
+  @Test
+  void trustedPackages() throws Exception {
+    Schema schema = new Schema.Parser().parse(new File("target/test-classes/schema-trusted-packages.json"));
+
+    Schema.Field field = schema.getField("value");
+    System.out.println(field);
+
+    System.out.println(schema);
```

Comment:
This test looks like a playground. 
It should be either finished (e.g. by using assertions) or removed.

Reply:
It should be removed (my bad sorry about that). I will clean up.